src/components/text-input/_text-input.scss

```diff
@@ -74,6 +73,35 @@
         display: none;
       }
     }
+
+    &-wrapper svg[hidden='true'] {
```

Comment:
The existence `hidden` attribute should be treated as hidden - So can we do: `svg[hidden]`?

Reply:
sure, I have refactored it so that it will remove the `hidden` attribute entirely when we want to show the icon

I was initially selecting `svg[hidden="true"]` here because I was toggling between `hidden="true"` and `hidden="false"`, but now I can select `svg[hidden]` since the attribute will either be set or removed